beta/src/pages/learn/conditional-rendering.md

```diff
@@ -63,7 +63,7 @@ if (isPacked) {
 return <li className="item">{name}</li>;
 ```
 
-If the `isPacked` prop is `true`, this code **returns a different JSX tree**. With this change, some of the items get a checkmark at the end:
+यदि `isPacked` prop `true` है, तो यह कोड **एक अलग JSX ट्री देता है**। इस परिवर्तन के साथ, कुछ वस्तुओं को अंत में एक चेकमार्क मिलता है:
```

Comment:
`वस्तुओं` => `चीज़ों`

I think `चीज़ों` makes more sense here?
Let me know your thoughts on this.

Reply:
so we are referring to `items` so i guess both word will work fine, `चीज़ों` is used more commonly. so it make sense to change to that.